src/compat/distributionsad.jl

```diff
@@ -0,0 +1,246 @@
+using .DistributionsAD: TuringWishart, TuringInverseWishart, MultipleContinuousUnivariate,
+                        MatrixContinuousUnivariate, MultipleContinuousMultivariate,
+                        ProductVectorContinuousUnivariate,
+                        ProductMatrixContinuousUnivariate,
+                        ProductVectorContinuousMultivariate
+
+# TuringWishart
+
+function logpdf_with_trans(
+    d::TuringWishart,
+    X::AbstractMatrix{<:Real},
+    transform::Bool
+)
+    _logpdf_with_trans_pd(d, X, transform)
+end
+function logpdf_with_trans(
+    d::TuringWishart,
+    X::AbstractArray{<:AbstractMatrix{<:Real}},
+    transform::Bool
+)
+    _logpdf_with_trans_pd.(Ref(d), X, transform)
+end
+function link(d::TuringWishart, X::AbstractMatrix{<:Real})
```

Comment:
Regarding _all_ these `link` and `invlink` definitions: would it maybe be a good idea to instead define a corresponding `Bijector` for these, and just let `link` and `invlink` be a proxy for evaluating this `Bijector` instead? This way we don't end up having to repeat ourselves when we inevitably implement these bijectors.

Reply:
Sounds good.